indra/newview/llagentcamera.cpp

```diff
@@ -1764,25 +1764,32 @@ LLVector3d LLAgentCamera::calcCameraPositionTargetGlobal(BOOL *hit_limit)
 			LL_WARNS() << "Null avatar drawable!" << LL_ENDL;
 			return LLVector3d::zero;
 		}
+
 		head_offset.clearVec();
+		F32 fixup;
+		if (gAgentAvatarp->hasPelvisFixup(fixup))
+		{
+			head_offset[VZ] -= fixup;
+		}
+		if (gAgentAvatarp->isSitting())
+		{
+			head_offset.mdV[VZ] += 0.1;
```

Comment:
This is confusing, why does it need to be adjusted and why by 0.1 specifically?

Reply:
This is an old expression
I only removed duplicated magic numbers